Makefile

```diff
@@ -0,0 +1,19 @@
+# Styling
```

Comment:
Maybe we could add a rule for Git hook also, i.e to install pre-commit hook and to run it on all files? That way it will be easier for the users to just use one rule to run all the formatting stuff defined in pre-commit-config.

Reply:
Sure, let me find a way to do that.

Reply:
Check this out:
https://gist.github.com/mmphego/dc933ba1bf9630fe57e46e0da5ef7820#file-makefile